### ISSUE TITLE
No images

### ISSUE THREAD

**Issue by fedwards01** [dvcoolarun/web2pdf#2]
I'm not sure if this is an "issue" or a "feature", but the PDFs are not including images from many webpages. Particularly Wikipedia pages.

**Comment by dvcoolarun** [comment]
Yes, Some pdf files the tool generates don't include images, so a dependency may be missing.

I'll need to look into it in more detail.

**Comment by rishavraj543256** [comment]
<img width="960" alt="error" src="https://github.com/user-attachments/assets/d61db208-26b2-4683-861c-59c86af77854" />
I am getting this error
Fontconfig error: Cannot load default config file: No such file: (null)
